index.html

```diff
@@ -128,24 +145,27 @@ <h2 class="about-header">About Myself</h2>
                        <li class="list-item"><img src="images/angelist.png" alt="follow me on angellist" width="20" height="20" ></li>
                    </ul>
                    <button class="btn-resume">Get my resume</button>
-                   <div class="flex-container">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style to all buttons as shown in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Uy7TAUKVJbGSY3Ol-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/213466408-afc9593c-5513-462f-a2c2-6193886a7161.png)